bindings/node/examples/documentation/quicktour.test.ts

```diff
@@ -155,20 +149,19 @@ describe("quicktourExample", () => {
         // END print_type_ids
         expect(output.getTypeIds()).toEqual([0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 1, 1]);
         // START encode_batch
-        let encodeBatch = promisify(tokenizer.encodeBatch.bind(tokenizer));
 
-        var output = await encodeBatch(["Hello, y'all!", "How are you 😁 ?"]);
+        var output = await tokenizer.encodeBatch(["Hello, y'all!", "How are you 😁 ?"]);
         // END encode_batch
         // START encode_batch_pair
-        var output = await encodeBatch(
-            [["Hello, y'all!", "How are you 😁 ?"], ["Hello to you too!", "I'm fine, thank you!"]]
-        );
+        // var output = await tokenizer.encodeBatch(
+        //     [["Hello, y'all!", "How are you 😁 ?"], ["Hello to you too!", "I'm fine, thank you!"]]
+        // );
```

Comment:
Yes I removed 1 feature:

PreTokenized inputs. Which are presplitted strings (so list of strings).

Why did I remove it :

- It bloats quite a lot the code (it would require rewriting the argument parsing since it doesn't work by default with the macro, or at least I wasn't able to).
- It's a rarely used feature which have lots of caveats.
- We can readd later, right now, running on more recent versions in reasonable time was the goal.